core/urls.py

```diff
@@ -11,4 +11,5 @@
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^admin/', admin.site.urls),
     url(r'^api$', csrf_exempt(authorization_required(GraphQLView.as_view()))),
+    url(r'^gh$', csrf_exempt(GraphQLView.as_view(graphiql=True))),
```

Comment:
@andela-snwuguru: I understand how frustrating it is for development. But I think we really need it there for both staging and production environments. I would suggest you use it like this in development, but do not commit it.